### PR TITLE
Updated python version from 3.6->3.9, and improved lambda function code

### PR DESCRIPTION
**Pull request by subashc2023**
Updated lambda function to use 'f strings' which have become best practice after 3.6
Updated lambda functions to use Pythons built-in logger module.
Updated Lambda function runtime to Python 3.9 from 3.6

### CODE REVIEW

**Comment by subashc2023** [comment]
upworkdemo
upworkdemo2

File: rds_autostart.yml [hunk 1]
```diff
@@ -231,7 +231,7 @@ Resources:
       Role: !GetAtt rdsAutoStopWorkerFunctionRole.Arn
       Timeout: 30
       Handler: index.handler
-      Runtime: python3.6
+      Runtime: python3.9
       Code:
         ZipFile: |
```
Comment (jimzucker) [inline rds_autostart.yml:236]: code not updated per descriptions, deployment will include this old code.  Dont see logger changes

File: rdsAutoStopLambda.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import json
```
Comment (jimzucker) [inline rdsAutoStopLambda.py]: Not sure why this file is here, if you want to have a 'Copy' for testing that's OK, lets put it in a folder src/xxx.py and add headers for the license and comment these are for testing and not used in the deployment

File: rdsAutoStopLambdaWorker.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import json
```
Comment (jimzucker) [inline rdsAutoStopLambdaWorker.py]: Not sure why this file is here, if you want to have a 'Copy' for testing that's OK, lets put it in a folder src/xxx.py and add headers for the license and comment these are for testing and not used in the deployment


code in yaml is what this project deploys

**Review by jimzucker — CHANGES_REQUESTED** [review]
For the video you should start with showing its running and he stopping works not take viewers time with the setup.  also needs narration

File: rds_autostart.yml (hunk 1 above)
Comment (subashc2023) [reply rds_autostart.yml:236]: Addressed

File: rdsAutoStopLambda.py (hunk 2 above)
Comment (subashc2023) [reply rdsAutoStopLambda.py]: addressed

File: rdsAutoStopLambdaWorker.py (hunk 3 above)
Comment (subashc2023) [reply rdsAutoStopLambdaWorker.py]: addressed

**Pull request closed without merging**